doc/index.rst

```diff
@@ -80,6 +89,7 @@ Amazon provides implementations of some common machine learning algortithms opti
     factorization_machines
     lda
     ntm
+    randomcutforest
```

Comment:
this also needs a corresponding file. or just remove the added line because earlier today I did try to make files for all the missing 1P algos in #575